### PR TITLE
post v7.1.0 release chores

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

closes: #XXXX

### Commit Message / Changelog Entry

```bash
type: commit message
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [x] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: .github/compatibility-test-matrices/release-v4.1.x/incentivized-transfer-chain-b.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "chain-a": ["release-v4.2.x", "v7.0.1", "v6.2.0", "v6.1.1", "v5.3.1", "v5.2.1", "v4.4.2", "v4.3.1", "v4.2.2", "v4.1.3"],
+  "chain-a": ["release-v4.2.x", "v7.1.0", "v7.0.1", "v6.2.0", "v6.1.1", "v5.3.1", "v5.2.1", "v4.4.2", "v4.3.1", "v4.2.2", "v4.1.3"],
```
Comment (damiannolan) [inline .github/compatibility-test-matrices/release-v4.1.x/incentivized-transfer-chain-b.json:2]: this file is has `release-v4.1.x` but `release-v4.2.x` is used in the json blob

**Review by damiannolan — APPROVED** [review]

**Review by DimitrisJim — APPROVED** [review]

File: .github/compatibility-test-matrices/release-v4.4.x/transfer-chain-a.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   ],
   "chain-b": [
     "release-v4.4.x",
+    "v7.1.0",
     "v7.0.1",
     "v6.2.0",
     "v6.1.1",
```
Comment (chatton) [inline .github/compatibility-test-matrices/release-v4.4.x/transfer-chain-a.json:10]: versions on separate lines make these diff reviews way easier, is it possible to format the other files so we can see the diffs this way? (I'd be happy with a follow up PR so as to not make this one more difficult to review!)

**Review by chatton — APPROVED** [review]
LGTM! Just one suggestion for a follow up PR

File: .github/compatibility-test-matrices/release-v4.4.x/transfer-chain-a.json (hunk 2 above)
Comment (DimitrisJim) [reply .github/compatibility-test-matrices/release-v4.4.x/transfer-chain-a.json:10]: This does the trick locally (ran from the compatibility folder):

```bash
find . -name "*.json" -type f | while IFS= read -r json_file; do
    jq '.' "$json_file" > temp.json && mv temp.json "$json_file"
done
```

Comment (crodriguezvega) [reply .github/compatibility-test-matrices/release-v4.4.x/transfer-chain-a.json:10]: Nice @DimitrisJim, flexing your scripting muscles! 💪 Would you like to open a PR with those changes?

Comment (DimitrisJim) [reply .github/compatibility-test-matrices/release-v4.4.x/transfer-chain-a.json:10]: easy peasy lemon squeezy

Comment (chatton) [reply .github/compatibility-test-matrices/release-v4.4.x/transfer-chain-a.json:10]: 🍋

**Pull request merged**